pkg/nmi/server/endpointpolicy.go

```diff
@@ -39,13 +39,13 @@ func ApplyEndpointRoutePolicy(podIP string, metadataIP string, metadataPort stri
 	if err != nil {
 		if endpointPolicyError, ok := err.(*endpointPolicyError); ok {
 			if endpointPolicyError.errType == InvalidOperation {
-				return fmt.Errorf("Get endpoint for Pod IP - %s. Error: %w", podIP, endpointPolicyError.err)
+				return fmt.Errorf("Error: %v", endpointPolicyError.err)
 			} else if endpointPolicyError.errType == NotFound {
-				klog.Infof("No applying action: no endpoint found for Pod IP - %s.", podIP)
+				klog.Infof("Route policy will not be applied for pod: %v", endpointPolicyError.err)
 				return nil
 			}
 		}
-		return fmt.Errorf("Get endpoint for Pod IP - %s. Error: %w", podIP, err)
+		return fmt.Errorf("Error: %v", err)
```

Comment:
>return fmt.Errorf("Error: %v", err) [](start = 2, length = 35)

These seems unnecessary. Since you are not adding anything new to the error, you might as well return err (same for ln 42).

Reply:
Thanks for explanation! I have updated it